# -*- coding: utf-8 -*-

#####################################################
# Copyright(C) @ 2023.                              #
# Authored by 太阳的小哥(bilibili)                    #
# Email: 1198017347@qq.com                          #
# CSDN: https://blog.csdn.net/qq_38412266?type=blog #
#####################################################


from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(1278, 1091)
        self.centralwidget = QtWidgets.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.verticalLayout_21 = QtWidgets.QVBoxLayout(self.centralwidget)
        self.verticalLayout_21.setObjectName("verticalLayout_21")
        self.horizontalLayout_13 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_13.setObjectName("horizontalLayout_13")
        self.groupBox = QtWidgets.QGroupBox(self.centralwidget)
        self.groupBox.setStyleSheet("border-color: rgb(0, 0, 0);")
        self.groupBox.setObjectName("groupBox")
        self.verticalLayout = QtWidgets.QVBoxLayout(self.groupBox)
        self.verticalLayout.setObjectName("verticalLayout")
        self.treeView = QtWidgets.QTreeView(self.groupBox)
        self.treeView.setObjectName("treeView")
        self.verticalLayout.addWidget(self.treeView)
        self.horizontalLayout_13.addWidget(self.groupBox)
        self.verticalLayout_19 = QtWidgets.QVBoxLayout()
        self.verticalLayout_19.setObjectName("verticalLayout_19")
        self.label = QtWidgets.QLabel(self.centralwidget)
        font = QtGui.QFont()
        font.setPointSize(25)
        self.label.setFont(font)
        self.label.setAlignment(QtCore.Qt.AlignCenter)
        self.label.setObjectName("label")
        self.verticalLayout_19.addWidget(self.label)
        self.groupBox_2 = QtWidgets.QGroupBox(self.centralwidget)
        self.groupBox_2.setObjectName("groupBox_2")
        self.verticalLayout_2 = QtWidgets.QVBoxLayout(self.groupBox_2)
        self.verticalLayout_2.setObjectName("verticalLayout_2")
        self.graphicsView = QtWidgets.QGraphicsView(self.groupBox_2)
        self.graphicsView.setStyleSheet("background-color: rgb(135, 135, 135);")
        self.graphicsView.setObjectName("graphicsView")
        self.verticalLayout_2.addWidget(self.graphicsView)
        self.verticalLayout_19.addWidget(self.groupBox_2)
        self.tabWidget = QtWidgets.QTabWidget(self.centralwidget)
        self.tabWidget.setObjectName("tabWidget")
        self.tab = QtWidgets.QWidget()
        self.tab.setObjectName("tab")
        self.verticalLayout_3 = QtWidgets.QVBoxLayout(self.tab)
        self.verticalLayout_3.setObjectName("verticalLayout_3")
        self.progressBar = QtWidgets.QProgressBar(self.tab)
        self.progressBar.setProperty("value", 24)
        self.progressBar.setObjectName("progressBar")
        self.verticalLayout_3.addWidget(self.progressBar)
        self.listWidget = QtWidgets.QListWidget(self.tab)
        self.listWidget.setObjectName("listWidget")
        self.verticalLayout_3.addWidget(self.listWidget)
        self.tabWidget.addTab(self.tab, "")
        self.tab_2 = QtWidgets.QWidget()
        self.tab_2.setObjectName("tab_2")
        self.verticalLayout_22 = QtWidgets.QVBoxLayout(self.tab_2)
        self.verticalLayout_22.setObjectName("verticalLayout_22")
        self.horizontalLayout_14 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_14.setObjectName("horizontalLayout_14")
        self.pushButton_3 = QtWidgets.QPushButton(self.tab_2)
        font = QtGui.QFont()
        font.setPointSize(12)
        self.pushButton_3.setFont(font)
        self.pushButton_3.setObjectName("pushButton_3")
        self.horizontalLayout_14.addWidget(self.pushButton_3)
        self.pushButton_4 = QtWidgets.QPushButton(self.tab_2)
        font = QtGui.QFont()
        font.setPointSize(12)
        self.pushButton_4.setFont(font)
        self.pushButton_4.setObjectName("pushButton_4")
        self.horizontalLayout_14.addWidget(self.pushButton_4)
        self.pushButton_5 = QtWidgets.QPushButton(self.tab_2)
        font = QtGui.QFont()
        font.setPointSize(12)
        self.pushButton_5.setFont(font)
        self.pushButton_5.setObjectName("pushButton_5")
        self.horizontalLayout_14.addWidget(self.pushButton_5)
        self.horizontalLayout_14.setStretch(0, 1)
        self.horizontalLayout_14.setStretch(1, 1)
        self.horizontalLayout_14.setStretch(2, 1)
        self.verticalLayout_22.addLayout(self.horizontalLayout_14)
        self.tabWidget.addTab(self.tab_2, "")
        self.verticalLayout_19.addWidget(self.tabWidget)
        self.verticalLayout_19.setStretch(0, 1)
        self.verticalLayout_19.setStretch(1, 9)
        self.verticalLayout_19.setStretch(2, 4)
        self.horizontalLayout_13.addLayout(self.verticalLayout_19)
        self.verticalLayout_18 = QtWidgets.QVBoxLayout()
        self.verticalLayout_18.setObjectName("verticalLayout_18")
        self.groupBox_3 = QtWidgets.QGroupBox(self.centralwidget)
        self.groupBox_3.setObjectName("groupBox_3")
        self.verticalLayout_4 = QtWidgets.QVBoxLayout(self.groupBox_3)
        self.verticalLayout_4.setObjectName("verticalLayout_4")
        self.checkBox = QtWidgets.QCheckBox(self.groupBox_3)
        self.checkBox.setObjectName("checkBox")
        self.verticalLayout_4.addWidget(self.checkBox)
        self.verticalLayout_18.addWidget(self.groupBox_3)
        self.groupBox_4 = QtWidgets.QGroupBox(self.centralwidget)
        self.groupBox_4.setObjectName("groupBox_4")
        self.verticalLayout_5 = QtWidgets.QVBoxLayout(self.groupBox_4)
        self.verticalLayout_5.setObjectName("verticalLayout_5")
        self.checkBox_2 = QtWidgets.QCheckBox(self.groupBox_4)
        self.checkBox_2.setObjectName("checkBox_2")
        self.verticalLayout_5.addWidget(self.checkBox_2)
        self.verticalLayout_18.addWidget(self.groupBox_4)
        self.groupBox_5 = QtWidgets.QGroupBox(self.centralwidget)
        self.groupBox_5.setObjectName("groupBox_5")
        self.verticalLayout_9 = QtWidgets.QVBoxLayout(self.groupBox_5)
        self.verticalLayout_9.setObjectName("verticalLayout_9")
        self.verticalLayout_7 = QtWidgets.QVBoxLayout()
        self.verticalLayout_7.setObjectName("verticalLayout_7")
        self.checkBox_3 = QtWidgets.QCheckBox(self.groupBox_5)
        self.checkBox_3.setObjectName("checkBox_3")
        self.verticalLayout_7.addWidget(self.checkBox_3)
        self.horizontalLayout = QtWidgets.QHBoxLayout()
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.label_2 = QtWidgets.QLabel(self.groupBox_5)
        self.label_2.setObjectName("label_2")
        self.horizontalLayout.addWidget(self.label_2)
        self.spinBox = QtWidgets.QSpinBox(self.groupBox_5)
        self.spinBox.setObjectName("spinBox")
        self.horizontalLayout.addWidget(self.spinBox)
        self.horizontalLayout.setStretch(0, 1)
        self.horizontalLayout.setStretch(1, 1)
        self.verticalLayout_7.addLayout(self.horizontalLayout)
        self.verticalLayout_7.setStretch(0, 1)
        self.verticalLayout_7.setStretch(1, 1)
        self.verticalLayout_9.addLayout(self.verticalLayout_7)
        self.verticalLayout_18.addWidget(self.groupBox_5)
        self.groupBox_11 = QtWidgets.QGroupBox(self.centralwidget)
        self.groupBox_11.setObjectName("groupBox_11")
        self.verticalLayout_20 = QtWidgets.QVBoxLayout(self.groupBox_11)
        self.verticalLayout_20.setObjectName("verticalLayout_20")
        self.verticalLayout_6 = QtWidgets.QVBoxLayout()
        self.verticalLayout_6.setObjectName("verticalLayout_6")
        self.checkBox_11 = QtWidgets.QCheckBox(self.groupBox_11)
        self.checkBox_11.setObjectName("checkBox_11")
        self.verticalLayout_6.addWidget(self.checkBox_11)
        self.horizontalLayout_12 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_12.setObjectName("horizontalLayout_12")
        self.label_12 = QtWidgets.QLabel(self.groupBox_11)
        self.label_12.setObjectName("label_12")
        self.horizontalLayout_12.addWidget(self.label_12)
        self.checkBox_13 = QtWidgets.QCheckBox(self.groupBox_11)
        self.checkBox_13.setText("")
        self.checkBox_13.setObjectName("checkBox_13")
        self.horizontalLayout_12.addWidget(self.checkBox_13)
        self.horizontalLayout_12.setStretch(0, 1)
        self.horizontalLayout_12.setStretch(1, 1)
        self.verticalLayout_6.addLayout(self.horizontalLayout_12)
        self.horizontalLayout_11 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_11.setObjectName("horizontalLayout_11")
        self.label_11 = QtWidgets.QLabel(self.groupBox_11)
        self.label_11.setObjectName("label_11")
        self.horizontalLayout_11.addWidget(self.label_11)
        self.checkBox_12 = QtWidgets.QCheckBox(self.groupBox_11)
        self.checkBox_12.setText("")
        self.checkBox_12.setObjectName("checkBox_12")
        self.horizontalLayout_11.addWidget(self.checkBox_12)
        self.horizontalLayout_11.setStretch(0, 1)
        self.horizontalLayout_11.setStretch(1, 1)
        self.verticalLayout_6.addLayout(self.horizontalLayout_11)
        self.verticalLayout_20.addLayout(self.verticalLayout_6)
        self.verticalLayout_18.addWidget(self.groupBox_11)
        self.groupBox_6 = QtWidgets.QGroupBox(self.centralwidget)
        self.groupBox_6.setObjectName("groupBox_6")
        self.verticalLayout_11 = QtWidgets.QVBoxLayout(self.groupBox_6)
        self.verticalLayout_11.setObjectName("verticalLayout_11")
        self.verticalLayout_10 = QtWidgets.QVBoxLayout()
        self.verticalLayout_10.setObjectName("verticalLayout_10")
        self.checkBox_4 = QtWidgets.QCheckBox(self.groupBox_6)
        self.checkBox_4.setObjectName("checkBox_4")
        self.verticalLayout_10.addWidget(self.checkBox_4)
        self.horizontalLayout_2 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_2.setObjectName("horizontalLayout_2")
        self.label_3 = QtWidgets.QLabel(self.groupBox_6)
        self.label_3.setObjectName("label_3")
        self.horizontalLayout_2.addWidget(self.label_3)
        self.horizontalSlider = QtWidgets.QSlider(self.groupBox_6)
        self.horizontalSlider.setOrientation(QtCore.Qt.Horizontal)
        self.horizontalSlider.setObjectName("horizontalSlider")
        self.horizontalLayout_2.addWidget(self.horizontalSlider)
        self.lineEdit = QtWidgets.QLineEdit(self.groupBox_6)
        self.lineEdit.setObjectName("lineEdit")
        self.horizontalLayout_2.addWidget(self.lineEdit)
        self.horizontalLayout_2.setStretch(0, 1)
        self.horizontalLayout_2.setStretch(1, 2)
        self.horizontalLayout_2.setStretch(2, 1)
        self.verticalLayout_10.addLayout(self.horizontalLayout_2)
        self.verticalLayout_10.setStretch(0, 1)
        self.verticalLayout_10.setStretch(1, 1)
        self.verticalLayout_11.addLayout(self.verticalLayout_10)
        self.verticalLayout_18.addWidget(self.groupBox_6)
        self.groupBox_7 = QtWidgets.QGroupBox(self.centralwidget)
        self.groupBox_7.setObjectName("groupBox_7")
        self.verticalLayout_12 = QtWidgets.QVBoxLayout(self.groupBox_7)
        self.verticalLayout_12.setObjectName("verticalLayout_12")
        self.verticalLayout_8 = QtWidgets.QVBoxLayout()
        self.verticalLayout_8.setObjectName("verticalLayout_8")
        self.checkBox_5 = QtWidgets.QCheckBox(self.groupBox_7)
        self.checkBox_5.setObjectName("checkBox_5")
        self.verticalLayout_8.addWidget(self.checkBox_5)
        self.horizontalLayout_3 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_3.setObjectName("horizontalLayout_3")
        self.label_4 = QtWidgets.QLabel(self.groupBox_7)
        self.label_4.setObjectName("label_4")
        self.horizontalLayout_3.addWidget(self.label_4)
        self.checkBox_6 = QtWidgets.QCheckBox(self.groupBox_7)
        self.checkBox_6.setText("")
        self.checkBox_6.setObjectName("checkBox_6")
        self.horizontalLayout_3.addWidget(self.checkBox_6)
        self.horizontalLayout_3.setStretch(0, 1)
        self.horizontalLayout_3.setStretch(1, 1)
        self.verticalLayout_8.addLayout(self.horizontalLayout_3)
        self.horizontalLayout_4 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_4.setObjectName("horizontalLayout_4")
        self.label_5 = QtWidgets.QLabel(self.groupBox_7)
        self.label_5.setObjectName("label_5")
        self.horizontalLayout_4.addWidget(self.label_5)
        self.checkBox_7 = QtWidgets.QCheckBox(self.groupBox_7)
        self.checkBox_7.setText("")
        self.checkBox_7.setObjectName("checkBox_7")
        self.horizontalLayout_4.addWidget(self.checkBox_7)
        self.horizontalLayout_4.setStretch(0, 1)
        self.horizontalLayout_4.setStretch(1, 1)
        self.verticalLayout_8.addLayout(self.horizontalLayout_4)
        self.horizontalLayout_5 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_5.setObjectName("horizontalLayout_5")
        self.label_6 = QtWidgets.QLabel(self.groupBox_7)
        self.label_6.setObjectName("label_6")
        self.horizontalLayout_5.addWidget(self.label_6)
        self.checkBox_8 = QtWidgets.QCheckBox(self.groupBox_7)
        self.checkBox_8.setText("")
        self.checkBox_8.setObjectName("checkBox_8")
        self.horizontalLayout_5.addWidget(self.checkBox_8)
        self.horizontalLayout_5.setStretch(0, 1)
        self.horizontalLayout_5.setStretch(1, 1)
        self.verticalLayout_8.addLayout(self.horizontalLayout_5)
        self.verticalLayout_8.setStretch(0, 1)
        self.verticalLayout_8.setStretch(1, 1)
        self.verticalLayout_8.setStretch(2, 1)
        self.verticalLayout_8.setStretch(3, 1)
        self.verticalLayout_12.addLayout(self.verticalLayout_8)
        self.verticalLayout_18.addWidget(self.groupBox_7)
        self.groupBox_8 = QtWidgets.QGroupBox(self.centralwidget)
        self.groupBox_8.setObjectName("groupBox_8")
        self.verticalLayout_14 = QtWidgets.QVBoxLayout(self.groupBox_8)
        self.verticalLayout_14.setObjectName("verticalLayout_14")
        self.verticalLayout_13 = QtWidgets.QVBoxLayout()
        self.verticalLayout_13.setObjectName("verticalLayout_13")
        self.checkBox_9 = QtWidgets.QCheckBox(self.groupBox_8)
        self.checkBox_9.setObjectName("checkBox_9")
        self.verticalLayout_13.addWidget(self.checkBox_9)
        self.horizontalLayout_6 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_6.setObjectName("horizontalLayout_6")
        self.label_7 = QtWidgets.QLabel(self.groupBox_8)
        self.label_7.setObjectName("label_7")
        self.horizontalLayout_6.addWidget(self.label_7)
        self.horizontalSlider_2 = QtWidgets.QSlider(self.groupBox_8)
        self.horizontalSlider_2.setOrientation(QtCore.Qt.Horizontal)
        self.horizontalSlider_2.setObjectName("horizontalSlider_2")
        self.horizontalLayout_6.addWidget(self.horizontalSlider_2)
        self.lineEdit_2 = QtWidgets.QLineEdit(self.groupBox_8)
        self.lineEdit_2.setObjectName("lineEdit_2")
        self.horizontalLayout_6.addWidget(self.lineEdit_2)
        self.horizontalLayout_6.setStretch(0, 1)
        self.horizontalLayout_6.setStretch(1, 2)
        self.horizontalLayout_6.setStretch(2, 1)
        self.verticalLayout_13.addLayout(self.horizontalLayout_6)
        self.horizontalLayout_7 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_7.setObjectName("horizontalLayout_7")
        self.label_8 = QtWidgets.QLabel(self.groupBox_8)
        self.label_8.setObjectName("label_8")
        self.horizontalLayout_7.addWidget(self.label_8)
        self.horizontalSlider_3 = QtWidgets.QSlider(self.groupBox_8)
        self.horizontalSlider_3.setOrientation(QtCore.Qt.Horizontal)
        self.horizontalSlider_3.setObjectName("horizontalSlider_3")
        self.horizontalLayout_7.addWidget(self.horizontalSlider_3)
        self.lineEdit_3 = QtWidgets.QLineEdit(self.groupBox_8)
        self.lineEdit_3.setObjectName("lineEdit_3")
        self.horizontalLayout_7.addWidget(self.lineEdit_3)
        self.horizontalLayout_7.setStretch(0, 1)
        self.horizontalLayout_7.setStretch(1, 2)
        self.horizontalLayout_7.setStretch(2, 1)
        self.verticalLayout_13.addLayout(self.horizontalLayout_7)
        self.verticalLayout_14.addLayout(self.verticalLayout_13)
        self.verticalLayout_18.addWidget(self.groupBox_8)
        self.groupBox_9 = QtWidgets.QGroupBox(self.centralwidget)
        self.groupBox_9.setObjectName("groupBox_9")
        self.verticalLayout_15 = QtWidgets.QVBoxLayout(self.groupBox_9)
        self.verticalLayout_15.setObjectName("verticalLayout_15")
        self.verticalLayout_16 = QtWidgets.QVBoxLayout()
        self.verticalLayout_16.setObjectName("verticalLayout_16")
        self.checkBox_10 = QtWidgets.QCheckBox(self.groupBox_9)
        self.checkBox_10.setObjectName("checkBox_10")
        self.verticalLayout_16.addWidget(self.checkBox_10)
        self.horizontalLayout_8 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_8.setObjectName("horizontalLayout_8")
        self.label_9 = QtWidgets.QLabel(self.groupBox_9)
        self.label_9.setObjectName("label_9")
        self.horizontalLayout_8.addWidget(self.label_9)
        self.horizontalSlider_4 = QtWidgets.QSlider(self.groupBox_9)
        self.horizontalSlider_4.setOrientation(QtCore.Qt.Horizontal)
        self.horizontalSlider_4.setObjectName("horizontalSlider_4")
        self.horizontalLayout_8.addWidget(self.horizontalSlider_4)
        self.lineEdit_4 = QtWidgets.QLineEdit(self.groupBox_9)
        self.lineEdit_4.setObjectName("lineEdit_4")
        self.horizontalLayout_8.addWidget(self.lineEdit_4)
        self.horizontalLayout_8.setStretch(0, 1)
        self.horizontalLayout_8.setStretch(1, 2)
        self.horizontalLayout_8.setStretch(2, 1)
        self.verticalLayout_16.addLayout(self.horizontalLayout_8)
        self.horizontalLayout_9 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_9.setObjectName("horizontalLayout_9")
        self.label_10 = QtWidgets.QLabel(self.groupBox_9)
        self.label_10.setObjectName("label_10")
        self.horizontalLayout_9.addWidget(self.label_10)
        self.horizontalSlider_5 = QtWidgets.QSlider(self.groupBox_9)
        self.horizontalSlider_5.setOrientation(QtCore.Qt.Horizontal)
        self.horizontalSlider_5.setObjectName("horizontalSlider_5")
        self.horizontalLayout_9.addWidget(self.horizontalSlider_5)
        self.lineEdit_5 = QtWidgets.QLineEdit(self.groupBox_9)
        self.lineEdit_5.setObjectName("lineEdit_5")
        self.horizontalLayout_9.addWidget(self.lineEdit_5)
        self.horizontalLayout_9.setStretch(0, 1)
        self.horizontalLayout_9.setStretch(1, 2)
        self.horizontalLayout_9.setStretch(2, 1)
        self.verticalLayout_16.addLayout(self.horizontalLayout_9)
        self.verticalLayout_15.addLayout(self.verticalLayout_16)
        self.verticalLayout_18.addWidget(self.groupBox_9)
        self.groupBox_10 = QtWidgets.QGroupBox(self.centralwidget)
        self.groupBox_10.setObjectName("groupBox_10")
        self.verticalLayout_17 = QtWidgets.QVBoxLayout(self.groupBox_10)
        self.verticalLayout_17.setObjectName("verticalLayout_17")
        self.textEdit = QtWidgets.QTextEdit(self.groupBox_10)
        font = QtGui.QFont()
        font.setPointSize(20)
        self.textEdit.setFont(font)
        self.textEdit.setObjectName("textEdit")
        self.verticalLayout_17.addWidget(self.textEdit)
        self.verticalLayout_18.addWidget(self.groupBox_10)
        self.horizontalLayout_10 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_10.setObjectName("horizontalLayout_10")
        self.pushButton = QtWidgets.QPushButton(self.centralwidget)
        font = QtGui.QFont()
        font.setPointSize(12)
        self.pushButton.setFont(font)
        self.pushButton.setObjectName("pushButton")
        self.horizontalLayout_10.addWidget(self.pushButton)
        self.pushButton_2 = QtWidgets.QPushButton(self.centralwidget)
        font = QtGui.QFont()
        font.setPointSize(12)
        self.pushButton_2.setFont(font)
        self.pushButton_2.setObjectName("pushButton_2")
        self.horizontalLayout_10.addWidget(self.pushButton_2)
        self.verticalLayout_18.addLayout(self.horizontalLayout_10)
        self.verticalLayout_18.setStretch(0, 1)
        self.verticalLayout_18.setStretch(1, 1)
        self.verticalLayout_18.setStretch(2, 2)
        self.verticalLayout_18.setStretch(3, 2)
        self.verticalLayout_18.setStretch(4, 2)
        self.verticalLayout_18.setStretch(5, 3)
        self.verticalLayout_18.setStretch(6, 3)
        self.verticalLayout_18.setStretch(7, 3)
        self.verticalLayout_18.setStretch(8, 2)
        self.verticalLayout_18.setStretch(9, 1)
        self.horizontalLayout_13.addLayout(self.verticalLayout_18)
        self.horizontalLayout_13.setStretch(0, 1)
        self.horizontalLayout_13.setStretch(1, 4)
        self.horizontalLayout_13.setStretch(2, 1)
        self.verticalLayout_21.addLayout(self.horizontalLayout_13)
        MainWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(MainWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 1278, 26))
        self.menubar.setObjectName("menubar")
        MainWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)

        self.retranslateUi(MainWindow)
        self.tabWidget.setCurrentIndex(0)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "皮肤病辅助诊断系统"))
        self.groupBox.setTitle(_translate("MainWindow", "文件浏览"))
        self.label.setText(_translate("MainWindow", "皮肤病辅助诊断系统"))
        self.groupBox_2.setTitle(_translate("MainWindow", "原图"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab), _translate("MainWindow", "缩略图"))
        self.pushButton_3.setText(_translate("MainWindow", "打开摄像头"))
        self.pushButton_4.setText(_translate("MainWindow", "关闭摄像头"))
        self.pushButton_5.setText(_translate("MainWindow", "截图"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab_2), _translate("MainWindow", "视频拍摄"))
        self.groupBox_3.setTitle(_translate("MainWindow", "毛发噪声"))
        self.checkBox.setText(_translate("MainWindow", "去除毛发噪声"))
        self.groupBox_4.setTitle(_translate("MainWindow", "肿瘤区域"))
        self.checkBox_2.setText(_translate("MainWindow", "分割肿瘤区域"))
        self.groupBox_5.setTitle(_translate("MainWindow", "平滑处理"))
        self.checkBox_3.setText(_translate("MainWindow", "进行平滑处理"))
        self.label_2.setText(_translate("MainWindow", "核大小"))
        self.groupBox_11.setTitle(_translate("MainWindow", "噪声"))
        self.checkBox_11.setText(_translate("MainWindow", "进行噪声处理"))
        self.label_12.setText(_translate("MainWindow", "椒盐噪声"))
        self.label_11.setText(_translate("MainWindow", "高斯噪声"))
        self.groupBox_6.setTitle(_translate("MainWindow", "伽马矫正"))
        self.checkBox_4.setText(_translate("MainWindow", "进行伽马矫正"))
        self.label_3.setText(_translate("MainWindow", "gamma值"))
        self.groupBox_7.setTitle(_translate("MainWindow", "均衡化"))
        self.checkBox_5.setText(_translate("MainWindow", "进行均衡化"))
        self.label_4.setText(_translate("MainWindow", "R通道"))
        self.label_5.setText(_translate("MainWindow", "G通道"))
        self.label_6.setText(_translate("MainWindow", "B通道"))
        self.groupBox_8.setTitle(_translate("MainWindow", "亮度调节"))
        self.checkBox_9.setText(_translate("MainWindow", "进行亮度调节"))
        self.label_7.setText(_translate("MainWindow", "alpha"))
        self.label_8.setText(_translate("MainWindow", "beta"))
        self.groupBox_9.setTitle(_translate("MainWindow", "边缘检测"))
        self.checkBox_10.setText(_translate("MainWindow", "进行边缘检测"))
        self.label_9.setText(_translate("MainWindow", "阈值1"))
        self.label_10.setText(_translate("MainWindow", "阈值2"))
        self.groupBox_10.setTitle(_translate("MainWindow", "分类结果"))
        self.pushButton.setText(_translate("MainWindow", "识别"))
        self.pushButton_2.setText(_translate("MainWindow", "查询"))
